dags/landsat_scenes_sync/landsat_scenes_gap_report.py

```diff
@@ -116,17 +115,15 @@ def build_path(file_row):
         for row in read_big_csv_files_from_gzip(file_path)
         if (
             # Filter to skip all LANDSAT_4
-            row.get("Satellite")
-            and row["Satellite"] != "LANDSAT_4"
+            row.get("Satellite") is not None and row["Satellite"] != "LANDSAT_4"
             # Filter to get just day
             and (
-                row.get("Day/Night Indicator")
+                row.get("Day/Night Indicator") is not None
```

Comment:
Why did you add **is not None**, here?

Reply:
https://stackoverflow.com/questions/3965104/not-none-test-in-python